app/scripts/controllers/preferences.test.js

```diff
@@ -274,20 +277,21 @@ describe('preferences controller', function () {
     });
   });
   describe('setUseTokenDetection', function () {
-    it('should default to false', function () {
+    process.env.TOKEN_DETECTION_V2 = true;
```

Comment:
We should move this into the test case itself, `it`, or in the `before`/`after` hooks. The logic directly placed into the `describe` block won't be part of the setup and teardown of the test suite itself.

```
  describe.only('Testing order of calls', function () {
    console.log('1 - describe');

    before(function () {
      return console.log('1 - before');
    });
    beforeEach(function () {
      return console.log('1 - beforeEach');
    });
    afterEach(function () {
      return console.log('1 - afterEach');
    });
    after(function () {
      return console.log('1 - after');
    });

    it('Test in describe 1', function () {
      return console.log('1 - test');
    });

    context('Scoped / Nested block', function () {
      console.log('2 - describe');

      it('Test in describe 2', function () {
        return console.log('2 - test');
      });
    });
  });
  ```
  will produce
  ```
1 - describe
2 - describe


  Transaction Controller
    Testing order of calls
1 - before
1 - beforeEach
1 - test
      ✓ Test in describe 1
1 - afterEach
      Scoped / Nested block
1 - beforeEach
2 - test
        ✓ Test in describe 2
1 - afterEach
1 - after
  ```